desktop/src/renderer/src/pages/transfers/LanTransfer.tsx

```diff
@@ -517,10 +517,16 @@ export const LanTransferPage = () => {
 
   useEffect(() => {
     if (!fileList) {
+      // Clear state.
+      setAccession(null);
+      setApplication(null);
+      setConfirmAccAppChecked(false);
       setFolders([]);
       setMetadata({});
+      return;
     }
-    parseFileList();
+    setProgressMade(!!fileList); // Track if progress is made.
+    parseFileList(); // Process the file list.
```

Comment:
Comment repeating the code
```suggestion
    parseFileList();
```